### PR TITLE
display info of the coin

### PR DESCRIPTION
**Pull request by marinanicolai**
@nicolad What method can I use to don't use so many useStates in the CoinPage component?

### CODE REVIEW

**Comment by marinanicolai** [comment]
@nicolad what am I declaring wrong in CoinInfo component, that is resulting in not displaying the data in CoinPage?

**Comment by nicolad** [comment]
please be more elaborate, what u tried to do, what are the errors, what steps u've taken to fix?

File: src/components/CoinInfo/CoinInfo.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { useQuery } from "react-query";
+import { useParams } from "react-router-dom";
+
+function CoinInfo() {
+  const coin = "";
+  const { isLoading, error, data } = useQuery("repoData", () =>
+    fetch(`https://api.coingecko.com/api/v3/coins/${coin}`).then((res) =>
+      res.json()
+    )
+  );
```
Comment (nicolad) [inline src/components/CoinInfo/CoinInfo.js:11]: how do u get `coin` here from params?

**Comment by nicolad** [comment]
https://react-hook-form.com/api/useformcontext/

**Pull request merged**